python/lsst/ts/ATWhiteLightSource/chillerModel.py

```diff
@@ -155,10 +155,13 @@ def __init__(self):
         }
 
         self.q = PriorityQueue()
+        self.disconnected = False 
         self.component = None
         self.realComponent = ChillerComponent()
```

Comment:
why having a real and fake component concurrently? you can probably set them dynamically at run time.